packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting-info/utilv2.js

```diff
@@ -72,12 +73,17 @@ describe('plugin-meetings', () => {
 
     describe('#getRequestBody', () => {
       it('for _PERSONAL_ROOM_', () => {
+        const webex = new WebexCore({
+          credentials: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyLCJyZWFsbSI6Im15LXJlYWxtIn0.U16gzUsaRW1VVikJA2VeXRHPX716tG1_B42oxzy1UMk'
```

Comment:
Where did these credentials come from? is this a real value?

Reply:
I got it from another test `packages/node_modules/@webex/webex-core/test/unit/spec/credentials/credentials.js` line 275

Reply:
I see, seems like an encoded base64 string for a JWT token. can you add a comment for that?